cmds/io/io.go

```diff
@@ -1,123 +1,122 @@
-// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Copyright 2010-2019 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// io lets you do IO operations.
+// io reads and writes to physical memory and ports.
 //
 // Synopsis:
```

Comment:
you want to indicate that they can be repeated.
io r 0x3f8 w 0x3f8 1 r 0x3f8

Reply:
Done!